### PR TITLE
fixes (#8254) Filters have no effect when applied before the experiment loads

### LINKED ISSUE #8254: Filling in a search query on Nimbus landing page before experiments load does not apply filter
If you start typing a search query, or possibly any other filter, before the experiments have finished loading, after they load they will not be filtered, so then you have to clear your filters and reapply them.  Maybe there's some way to apply the filters on the first load?

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3131)

### PR DESCRIPTION
**Pull request by cherish2003**
### Because
-  Filters have no effect when applied before the experiment loads
### This commit
- Disabled the filters untill the all the experiments loads which resloves this issue 
- Fixes #8254 
### Screenshots
Screenshot from 2023-04-01 22-00-35
- As you can see the filters are disabled untill the experiments loads and gets enabled after loaded

### CODE REVIEW

**Comment by cherish2003** [comment]
@yashikakhurana kindly review the code let me know any changes are required .Thank you

File: experimenter/experimenter/nimbus-ui/src/components/PageHome/FilterBar/index.tsx [hunk 1]
```diff
@@ -130,7 +133,10 @@ const FilterSelect = <
 
   const fieldValue = filterValue[filterValueName];
   const { applications } = useConfig();
-
+  const { loading } = useQuery<{
+    experiments: getAllExperiments_experiments[];
+  }>(GET_EXPERIMENTS_QUERY, { fetchPolicy: "network-only" });
+  console.log(loading);
```
Comment (yashikakhurana) [inline experimenter/experimenter/nimbus-ui/src/components/PageHome/FilterBar/index.tsx:139]: Can you please remove this log?

**Review by yashikakhurana — CHANGES_REQUESTED** [review]
tested it, working 💯 , just if you can remove the log statement, thank you @cherish2003 🎉

File: experimenter/experimenter/nimbus-ui/src/components/PageHome/FilterBar/index.tsx (hunk 1 above)
Comment (cherish2003) [reply experimenter/experimenter/nimbus-ui/src/components/PageHome/FilterBar/index.tsx]: @yashikakhurana i have removed it

**Review by yashikakhurana — APPROVED** [review]
thank you @cherish2003 👏

**Pull request merged**